File: minijax-examples/minijax-example-hello/src/test/java/com/example/HelloTest.java
package com.example;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;
import org.minijax.rs.test.MinijaxTest;

class HelloTest extends MinijaxTest {

    @BeforeEach
    public void setUp() {
        register(Hello.class);
    }

    @Test
    void testHello() {
        assertEquals("Hello world!", target("/").request().get(String.class));
    }
}

File: minijax-examples/minijax-example-kotlin/src/test/kotlin/demo/HelloTest.kt
package demo

import kotlin.test.assertEquals
import org.junit.Test
import org.minijax.test.MinijaxTest

class HelloTest : MinijaxTest() {

    @Test
    fun testAssert() {
        register(Hello::class.java)
        assertEquals("Hello world!", MinijaxTest.target("/").request().get(String::class.java))
    }
}
